dicom_utils/container/record.py

```diff
@@ -749,16 +749,23 @@ def has_implant(self) -> bool:
         return self.BreastImplantPresent == "YES"
 
     @classmethod
-    def is_complete_mammo_case(cls, records: Iterable["MammogramFileRecord"]) -> bool:
-        needed_views: Dict[MammogramView, Optional[MammogramFileRecord]] = {k: None for k in STANDARD_MAMMO_VIEWS}
+    def get_standard_mammo_view_lookup(
+        cls, records: Iterable["MammogramFileRecord"]
+    ) -> Dict[MammogramView, List["MammogramFileRecord"]]:
+        needed_views: Dict[MammogramView, List[MammogramFileRecord]] = {k: [] for k in STANDARD_MAMMO_VIEWS}
         for rec in records:
             # only consider standard views
             if not isinstance(rec, MammogramFileRecord) or not rec.is_standard_mammo_view:
                 continue
             key = rec.mammogram_view
             if key in needed_views:
-                needed_views[key] = rec
-        return all(needed_views.values())
+                needed_views[key].append(rec)
+        return {k: v for k, v in needed_views.items() if v}
```

Comment:
Could we use a `defaultdict` (defaulting to `list`) and simplify both the first and last lines of this function? Something like:
```python
needed_views = defaultdict(list)
...
return needed_views
```